src/playMiddleware.js

```diff
@@ -0,0 +1,60 @@
+import videojs from 'video.js';
+
+const obj = {};
+let videojsReference = videojs;
```

Comment:
A comment for this line would be helpful. It's not obvious why this is necessary.